README.md

```diff
@@ -63,25 +54,24 @@ dsl {
 }
 ```
 
-This will add a task `generatejava` under the group `omero-dsl` in gradle _(to print a list of available gradle
-tasks run `./gradlew tasks` in a terminal)_.
+This will add a task `generateJava` under the group `omero-dsl` in gradle _(to print a list of available gradle
```

Comment:
Should this simply be a separate sentence rather than an italicized parenthetical note?

Reply:
Updating.